quiche/src/lib.rs

```diff
@@ -5647,9 +5657,9 @@ impl Connection {
     ///
     /// If this returns true, the connection object cannot yet be dropped, but
     /// no new application data can be sent or received. An application should
-    /// continue calling the [`recv()`], [`send()`], [`timeout()`], and
-    /// [`on_timeout()`] methods as normal, until the [`is_closed()`] method
-    /// returns `true`.
+    /// continue calling the [`recv()`], [`timeout()`], and [`on_timeout()`]
+    /// methods as normal, until the [`is_closed()`] method returns `true`.
+    /// It is no longer necessary to call [`send()`].
```

Comment:
Does this rewording match your intention?

```suggestion
    /// In contrast, once `is_draining()` returns true, calling [`send()`] is not
    /// required because no new outgoing packets will be generated.
```